### DISCUSSION TITLE
Infinispan outdated topology warning message

### DISCUSSION THREAD

**Discussion by stevejr** [infinispan/infinispan#10530] [Q&A]
Hi,

I am seeing the following log message in my Infinispan pods and wondered if someone could confirm what causes it:

```
2022-12-07 13:22:22,777 WARN  (non-blocking-thread--p2-t2) [org.infinispan.server.hotrod.Encoder2x] ISPN028029: Client /172.17.0.8:11222 keeps providing outdated topology 1383681721
```

Looking at the code it says it should reset the topology id to stop the message being repeated but I am seeing it over and over again.

Thanks.

**Comment by wburns** [comment]
What client version (including minor) and server version are you using?

**Reply by stevejr** [reply]
The server is on Docker image tag 14.0.3.Final.

Not sure what you mean by client - the connection to the cluster is from a KeyCloak pod.

**Reply by wburns** [reply]
Keycloak will be using a specific version of the Infinispan client.

The client prints out a message on startup that says the version. Looks like `Infinispan version: <version>`. Are you able to obtain that?

**Comment by wburns** [comment]
This is caused from https://issues.redhat.com/browse/ISPN-13813, upgrading to 13.0.14+ when using a 14 server should remedy it.
